docs/Manage.md

```diff
@@ -71,7 +71,7 @@ web VMs through the specified file server (Gluster or NFS), and this is
 not good for web response time. Therefore, we introduced the
 `htmlLocalCopySwitch` that'll copy the `/azlamp/html` directory to
 `/var/www/html` in each autoscaled web VM and reconfigures the web
-server (apache/nginx)'s server root directory accordingly, when it's set
+server (nginx)'s server root directory accordingly, when it's set
```

Comment:
Has Apache been removed? Why? Many customers prefer Apache, especially for PHP workloads.